packages/documentation/static/code-runtime/html/src/index.html

```diff
@@ -1,20 +1 @@
-<!--
-SPDX-FileCopyrightText: 2023 Siemens AG
-
-SPDX-License-Identifier: MIT
-
-This source code is licensed under the MIT license found in the
-LICENSE file in the root directory of this source tree.
--->
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite App</title>
-  </head>
-  <body>
-    <!-- IX_INJECT_SOURCE_CODE -->
-    <script type="module" src="./main.js"></script>
-  </body>
-</html>
+<!-- IX_INJECT_SOURCE_CODE -->
```

Comment:
Why is it necessary to replace all the content? Please do not modify the runtime files in that way its runnable anymore.